### ISSUE TITLE
CSS conflicts with vue

### ISSUE THREAD

**Issue by raulvc** [mccabiles/vuetify-number-input#1]
I scaffolded a new project using the basic template and added this component to a dialog;
This is what it looks like:
![image](https://user-images.githubusercontent.com/2997449/58770052-e3774d00-8582-11e9-817b-6c3da2ec3b7f.png)

any ideas?

**Comment by MrJmpl3** [comment]
Are you sure, if import the Material Icons?

**Comment by raulvc** [comment]
yeah, at least 3 months ago it worked when outside v-dialogs and broke inside... 
I ended up using v-currency-field back then

**Comment by MrJmpl3** [comment]
mmm, it's illogical that it broke because the component doesn't generate or edit any CSS

**Comment by swivelsoftware-hong** [comment]
having same problem 
![image](https://user-images.githubusercontent.com/85144635/129509705-73f42cc1-1eb8-4185-8a25-789bbac6c29d.png)
where the icons doesn't display
